distributed/tests/test_steal.py

```diff
@@ -650,6 +655,9 @@ class Foo:
     assert not s.who_has
     assert not any(s.has_what.values())
 
+    import gc
```

Comment:
Should this be removed?

Reply:
Ah yes, thanks! I added that to try and fix this test, but it didn't do the trick